app/views/organizer_position_invites/_admin_signee_check.html.erb

```diff
@@ -0,0 +1,27 @@
+<div class="field field--select w-100"
```

Comment:
Let's add a strict locals comment

https://blog.appsignal.com/2024/09/11/ruby-on-rails-7-1-partial-strict-locals-and-their-gotchas.html